Mathlib/Analysis/SpecialFunctions/Pow/NNReal.lean

```diff
@@ -504,6 +504,10 @@ theorem coe_rpow_def (x : ℝ≥0) (y : ℝ) :
     (x : ℝ≥0∞) ^ y = if x = 0 ∧ y < 0 then ⊤ else ↑(x ^ y) :=
   rfl
 
+theorem rpow_ofNNReal {M : ℝ≥0} {P : ℝ} (hP : 0 ≤ P) :
+    (ENNReal.ofNNReal M).rpow P = ENNReal.ofNNReal (M ^ P) := by
```

Comment:
Can you please use the coercion here (via type ascription) to make this easier to read?